pkg/manifests/csv.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: packageserver
   namespace: openshift-operator-lifecycle-manager
   labels:
-    olm.version: 0.19.0
+    olm.version: 0.0.0-99ea6d7f94caf2b8bbc3b4dc579884089e06abcf
```

Comment:
This doesn't look right, and I think it will result in a CI failure.

Reply:
`make verify` also does `make verify-manifests`, and also does a bunch of other checks

Reply:
Unfortunately, the documentation I wrote (and the scripts, darn it!) do not include this information.